source/includes/_invoice-payment.md

```diff
@@ -589,54 +393,7 @@ invoicePaymentApi.create_chargeback(payment_id,
 
 > Example Response:
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 201 Created
-< Location: http://127.0.0.1:8080/1.0/kb/invoicePayments/cc7fcd4d-e701-4679-9741-41289103db83/
-< Content-Type: application/json
-< Content-Length: 0
-```
-
-```java
-class InvoicePayment {
-    org.killbill.billing.client.model.gen.InvoicePayment@e920eb58
-    targetInvoiceId: 84ae9d9a-badf-45c2-b53b-7b8a077e42ea
-    accountId: 4dd59373-be7a-409e-beaa-a17e8b234591
-    paymentId: 96930ff3-82e1-4556-888a-0cb07ec120d6
-    paymentNumber: 1
-    paymentExternalKey: 96930ff3-82e1-4556-888a-0cb07ec120d6
-    authAmount: 0
-    capturedAmount: 0
-    purchasedAmount: 0.00
-    refundedAmount: 0
-    creditedAmount: 0
-    currency: USD
-    paymentMethodId: a9408abd-d505-4462-afe6-0d29a202d8b3
-    transactions: [class PaymentTransaction {
-        org.killbill.billing.client.model.gen.PaymentTransaction@e72668f3
-        transactionId: 583ba30d-c6f4-4600-934e-73ef9782e444
-        transactionExternalKey: 583ba30d-c6f4-4600-934e-73ef9782e444
-        paymentId: 96930ff3-82e1-4556-888a-0cb07ec120d6
-        paymentExternalKey: 96930ff3-82e1-4556-888a-0cb07ec120d6
-        transactionType: PURCHASE
-        amount: 249.95
-        currency: USD
-        effectiveDate: 2012-09-26T00:00:08.000Z
-        processedAmount: 249.95
-        processedCurrency: USD
-        status: SUCCESS
-        gatewayErrorCode: 
-        gatewayErrorMsg: 
-        firstPaymentReferenceId: null
-        secondPaymentReferenceId: null
-        properties: null
-        auditLogs: []
-    }]
-    paymentAttempts: null
-    auditLogs: []
-}
-```
-```ruby
+```json
 {
    "targetInvoiceId":"dd185d1c-a4c5-4420-b06a-df42af446975",
    "accountId":"bb43a670-c644-4121-a981-ba5f5dac3b94",
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?